AndroidRunner/Adb.py

```diff
@@ -139,3 +140,15 @@ def logcat(device_id, regex=None):
         params += ' -e %s' % regex
     adb.set_target_by_name(device_id)
     return adb.get_logcat(lcfilter=params)
+
+
+def cleanup(cmd):
+    from time import sleep
+    if cmd == "restart":
```

Comment:
Here we should consider also the `terminate` case (and add it in the readme file). Basically, it does exactly the same as the body of this `if` statement, but without lines 152-154.